.github/contribution/testing-conventions.md

```diff
@@ -235,32 +235,42 @@ This part of testing is often abandoned by people, but we are aware of how impor
 
 <!-- AUTO-GENERATED-CONTENT:START (SUPPORTED_BROWSERS) -->
 
-- Chrome for Android 114
-- Chrome 114
-- Chrome 113
-- Chrome 112
-- Chrome 111
+- Chrome for Android 128
```

Comment:
(I didn't confirm but my best guess is) because these MD files are generated on build, so when it was generated there the inheritance from browserslistrc generated a different output 🤷